### ISSUE TITLE
use manifest.yml in combination with the Maven plugin

### ISSUE THREAD

**Issue by tonvanbart** [wdxxs2z/cloudfoundry-java-client-expend#1]
More of a question than an issue maybe: is it possible to use the Maven plugin, but still use the manifest.yml instead of configuration inside the plugin? This would be helpful in some of our usage scenarios.

**Comment by rubensgomes** [comment]
Any updates on the manifest.yml question?  

Thank you,  

Rubens

**Comment by tonvanbart** [comment]
Not really. We ended up just using Maven to put the wars we want to deploy in a known location, and use Cloud Foundry tooling (cf command) to take it from there.

**Comment by cptstffn** [comment]
Any updates or news to the question?
